src/console_implementation/main_header.hpp

```diff
@@ -5,16 +5,21 @@
 #include <memory>
 #include <thread>
 #include <chrono>
+#include <string>
+#include <fstream>
+#include <sstream>
 
 //Library definitions
 #include "../business_logic/include/Tplayer.hpp"
```

Comment:
Thanks to CMake integration there is no need for relative paths.
```suggestion
#include "Tplayer.hpp"
```